content/_index.md

```diff
@@ -3,4 +3,36 @@ title = "Stephen Augustus"
 tagline = "Caesar of Systems"
 +++
 
-Friendly neighborhood Kubernetes guy at VMware. Previously Red Hat and CoreOS. Proud geek. Black engineer. Made in NYC.
\ No newline at end of file
+Stephen is a Black engineering director and leader in open source communities.
+
+He currently serves the Kubernetes community as a Chair for
+[SIG Release][sig-release], a Lead for [WG Naming][wg-naming], and an owner of
+[Enhancements][enhancements] subproject.
+
+Additionally, he is one of the Chairs for
+[CNCF SIG Contributor Strategy][contrib-strat] and a [Dex][dex] maintainer.
+
+Since 2020, Stephen has the pleasure of being one of the Program Chairs for
+[KubeCon / CloudNativeCon][kubecon], the cloud native community's flagship
+conference.
+
+In 2020, Stephen also co-founded the [Inclusive Naming Initiative][ini], a
+cross-industry group dedicated to helping projects and companies make
+consistent, responsible choices to remove harmful language across codebases,
+standards, and documentation. He leads multiple workstreams here and maintains
+the initiative's infrastructure.
+
+In 2021, he joined Cisco as their first Head of Open Source within the Emerging
+Technologies & Incubation division.
```

Comment:
All of these paragraphs start the same way (In [date]...) ; it's repetitive. Try:

```suggestion
Stephen was one of the Program Chairs for
[KubeCon / CloudNativeCon][kubecon], the cloud native community's flagship
conference, in 2020 and 2021. 

Stephen also co-founded the [Inclusive Naming Initiative][ini] in 2020, a
cross-industry group dedicated to helping projects and companies make
consistent, responsible choices to remove harmful language across codebases,
standards, and documentation. He leads multiple workstreams here and maintains
the initiative's infrastructure.

He is currently at Cisco, as their first Head of Open Source within the Emerging
Technologies & Incubation division. 
```